source/_components/sensor.paloalto.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: "Palo Alto Sensor"
+description: "Palo Alto Device integration into Home Assistant."
+date: 2018-03-04 16:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: System Monitor
+ha_iot_class: "Local Polling"
+logo: palo_alto.png
+ha_release: 0.65
+---
+
+
+The `paloalto` sensor platform is displaying the statistical summary of a [Palo Alto](https://www.paloaltonetworks.com/) device.
+
+To enable this sensor, add the following lines to your `configuration.yaml` file for a GET request:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: paloalto
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration